ledger/participant-integration-api/src/test/lib/scala/platform/indexer/ha/EndlessReadService.scala

```diff
@@ -63,60 +62,59 @@ case class EndlessReadService(
     */
   override def stateUpdates(
       beginAfter: Option[Offset]
-  )(implicit loggingContext: LoggingContext): Source[(Offset, Update), NotUsed] =
-    synchronized {
-      logger.info(s"EndlessReadService.stateUpdates($beginAfter) called")
-      stateUpdatesCalls.incrementAndGet()
-      val startIndex: Int = beginAfter.map(index).getOrElse(0) + 1
-      Source
-        .fromIterator(() => Iterator.from(startIndex))
-        .throttle(updatesPerSecond, 1.second)
```

Comment:
Is removing throttling from here intentional? I can see that we can provide throttling via `additionalSourceFlow` but I cannot see it being used anywhere.

Reply:
I did not want to have throttling for the new IndexerBenchmark while I kept it for the previous use in:

https://github.com/digital-asset/daml/pull/13144/files#diff-563bd39bc14c221f0e991e1363f2e22042c0bb51ab277c63e5c2d3d9667b7e5bR93

Reply:
Ok, I see 👍 
In this case I would make the users of the `stateUpdates()` method responsible for wiring up throttling on their own by removing the `additionalSourceFlow` parameter completely and adding `.via(<throttling logic>)` at the site of use.

Reply:
This makes sense, thanks 👍

Reply:
> In fact I am not in particular happy with the IndexerStabilityTestFixture and I would challenge that to reimplement cleanly in a separate PR.

@skisel-da Can you please elaborate what `not happy` means to you? I mean:
* if we have a bug we need to fix it right away
* but if it means you have some beautification/aesthetics improvement refactoring in mind, and if that is a priority (cc @mziolekda  ), I would suggest to first understand what it is testing (HA indexer), how, and why. I would be happy to help to give you pointers, also @rautenrieth-da is an expert of that code. It would be a great new joiner learning project :+1:

Reply:
1. elaborated on `not happy` concern below in the other comment
2. sure, will do so, thanks